Description: Change the name of the executable script in the module.
 ... to support the Debian convention of having the script and the identically
 named module in the same directory.
From: David Steele <dsteele@gmail.com>

--- a/setup.py
+++ b/setup.py
@@ -27,7 +27,7 @@
     packages=find_packages(exclude=['ez_setup']),
     entry_points = {
         'console_scripts': [
-            'cloudprint = cloudprint.cloudprint:main',
+            'cloudprint-cmd = cloudprint.cloudprint:main',
         ],
     },
 )
